tutorials/cp-kyma-create-function-cli/cp-kyma-create-function-cli.md

```diff
@@ -114,56 +121,92 @@ You should see a version number.
           noAuth: true
     ```
 
-3. Deploy your API Rule. 
+3. Deploy your APIRule. 
    
-    ```bash
+    ```bash/Powershell
     kubectl apply -f "{PATH_TO_YOUR_CONFIG_FILE}"
     ```
 
     For example:   
 
-    ```bash
+    ```bash/Powershell
     kubectl apply -f "C:\tools\myapirule.yaml"
     ```
    
 ### Verify the Function exposure
 
+[OPTION BEGIN [macOS]]
+
 1. Run the following command to get the domain name of your Kyma cluster:
 
     ```bash
-    kubectl get gateway -n kyma-system kyma-gateway \
-            -o jsonpath='{.spec.servers[0].hosts[0]}'
+    kubectl get gateway -n kyma-system kyma-gateway -o jsonpath='{.spec.servers[0].hosts[0]}'
     ```
 
-    You should see output similar to this:
+    You see output similar to this:
     
     ```bash
     *.12345678.kyma.ondemand.com
     ```
 
 2. Export the result without the leading `*.` as an environment variable:
 
+
     ```bash
     export CLUSTER_DOMAIN={DOMAIN_NAME}
     ```
-
+    
     For example:
 
     ```bash
     export CLUSTER_DOMAIN=12345678.kyma.ondemand.com
     ```
 
 
-3. Run the following curl command, replacing `{CLUSTER_DOMAIN}` with your domain: 
+3. Run the following curl command: 
 
     ```bash
     curl https://hello-host.$CLUSTER_DOMAIN
     ```
 
+
+If the deployment was successful, you see the `Hello World!` message.
+
+[OPTION END]
+
+[OPTION BEGIN [Windows]]
+
+1. Run the following command to get the domain name of your Kyma cluster:
+
+    ```Powershell
+    kubectl get gateway -n kyma-system kyma-gateway -o jsonpath='{.spec.servers[0].hosts[0]}'
+    ```
+
+    You see output similar to this:
+    
+    ```Powershell
+    *.12345678.kyma.ondemand.com
+    ```
+
+2. Export the result without the leading `*.` as an environment variable:
+
+    ```Powershell
+    $Env:CLUSTER_DOMAIN="c-5fccac8.stage.kyma.ondemand.com"
+    ```
+    
```

Comment:
This should probably be deleted?
<img width="838" height="364" alt="Screenshot 2025-08-13 at 10 33 59" src="https://github.com/user-attachments/assets/40158660-f008-486f-9d3c-04f389471af9" />
```suggestion    
```

Reply:
🤦‍♀️ yes, I'll add a placeholder